pkgs/top-level/python-packages.nix

```
@@ -1347,6 +1349,8 @@ in {
 
   cppy = callPackage ../development/python-modules/cppy { };
 
+  cPyparsing = callPackage ../development/python-modules/cPyparsing { };
```

```suggestion
  cpyparsing = callPackage ../development/python-modules/cpyparsing { };
```
Please normalize the attribute name. According to the [python contributing guidelines](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines), package names within nix should follow [PEP 0503](https://www.python.org/dev/peps/pep-0503/#normalized-names)